functions/Set-DbaLogin.ps1

```diff
@@ -253,8 +270,20 @@ function Set-DbaLogin {
                 # Change the password
                 if (Test-bound -ParameterName 'SecurePassword') {
                     try {
+                        # these settings are required for must_change and are validated above
+                        if (Test-Bound MustChange) {
+                            Write-Message -Message "Setting check_policy and check_expiration to true for $l because the must_change option requires it." -Level Warning
+                            $l.PasswordPolicyEnforced = $PasswordPolicyEnforced
+                            $l.PasswordExpirationEnabled = $PasswordExpirationEnabled
+                            $l.Alter()
```

Comment:
This can cause unintended changes that may not be desired.

Reply:
The MustChange is meant for a specific use-case, we are not accounting for the other use case in the command. An issue needs to be opened for any bug surrounding the parameter as-is to better discuss how to address it.